src/main/java/org/example/tms/controller/CommentController.java

```diff
@@ -41,11 +44,12 @@ public class CommentController {
             example = "123e4567-e89b-12d3-a456-426614174000"
     )
```

Comment:
better provide openapi.yaml and work with it, not to pass swagger annotation, code looks not clean